src/node_kernels/BLTGammaM2015NodeKernel.C

```diff
@@ -157,8 +179,8 @@ BLTGammaM2015NodeKernel::execute(
   fonset1 = Rev / 2.2 / Re0c;
   fonset2 = stk::math::min(fonset1, 2.0);
   rt = density*tke/sdr/visc;
-  fonset3 = stk::math::max(1.0 - 0.0233236151603499 * rt * rt * rt, 0.0);
-  fonset = stk::math::max(fonset2 - fonset3, 0.0);
+  fonset3 = BLTmax(0.0, 1.0 - 0.0233236151603499 * rt * rt * rt);
```

Comment:
There are a lot of constants in here, do they need to be abstracted to SolutionOptions?

Reply:
I assume you are referring to the use of constants like 2.2, 2.0, 0.0233236151603499, etc.? If so, is there an advantage to abstracting them in SolutionOptions? Those values aren't turbulence modeling constants in the sense of say sigma_k in the k-omega equations. They are presented as ordinary floating numbers in the Menter (2015) publication, as opposed to them presented using symbols. I am open to your suggestions.

Reply:
Yeah, they probably don't belong in `SolutionOptions` then.  You could move them to local variables and put them in the constructor/setup or in the header file, but this is also probably fine.

Let's see if anyone else wants to weigh in on this.

Reply:
@jamelvin @psakievich Please see my reply below. I will define this numbers using **constexpr double** in the setup.